Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -219,15 +271,10 @@ public class XPCServer {
     
     private func handleEvent(connection: xpc_connection_t, event: xpc_object_t) {
         if xpc_get_type(event) == XPC_TYPE_DICTIONARY {
```

Comment:
Another case where my guard clause instinct tingles:


```swift
    private func handleEvent(connection: xpc_connection_t, event: xpc_object_t) {
        // Only dictionary types (and not anything which will be interpreted as an error) should be checked for
        // message acceptance as errors generated by libxpc can fail to meet the acceptor's criteria
        guard xpc_get_type(event) == XPC_TYPE_DICTIONARY else {
            self.errorHandler?(XPCError.fromXPCObject(event))
            return
        }

        guard self.messageAcceptor.acceptMessage(connection: connection, message: event) else {
            self.errorHandler?(.insecure)
            return
        }

        handleMessage(connection: connection, message: event)
    }
```

Reply:
Hmm maybe it's just my relative lack of usage of `guard`, but I find this a bit harder to read.

Reply:
On second thought, I do prefer this. Have updated the comment to make a bit more sense with this control flow.